clarity-ext-scripts/clarity_ext_scripts/covid/parse_pcr.py

```diff
@@ -40,6 +43,7 @@ def execute(self):
             original_sample.udf_map.force("CT source", output.api_resource.uri)
             self.context.update(original_sample)
 
+        ct_analyes_service.execute()
```

Comment:
Is there a typo here? s/ct_analyes/ct_analytes ?

Reply:
yes